src/object.cr

```diff
@@ -201,6 +201,23 @@ class Object
     yield self
   end
 
+  # Returns `true` if `self` is included in the *collection* argument.
+  #
+  # ```
+  # 10.in?(0..100)     # => true
+  # 10.in?({0, 1, 10}) # => true
+  # 10.in?(0, 1, 10)   # => true
+  # 10.in?(:foo, :bar) # => false
+  # ```
+  def in?(collection) : Bool
+    collection.includes?(self)
+  end
+
+  # :ditto:
+  def in?(*values : Object) : Bool
```

Comment:
I am curious. Wouldn't everything inherit from `Object`?
Would this ever constrain the types that this could call?

Reply:
You seems to have missed https://github.com/crystal-lang/crystal/pull/8720#discussion_r371924639